test/run/ch-run_misc.bats

```diff
@@ -33,6 +65,13 @@ EOF
     ch-run -w "$ch_timg" rm write
 }
 
+@test 'run image by name' {
```

Comment:
I think “`specify storage`” can likely be eliminated in favor of expanding this test to validate all three methods of finding storage directory location. You can use the verbose output to check where `ch-run` thinks the storage directory is.